cmd/jb/main.go

```diff
@@ -73,7 +63,7 @@ func Main() int {
 	initCmd := a.Command(initActionName, "Initialize a new empty jsonnetfile")
 
 	installCmd := a.Command(installActionName, "Install all dependencies or install specific ones")
-	installCmdURLs := installCmd.Arg("packages", "URLs to package to install").URLList()
+	installCmdPaths := installCmd.Arg("paths", "paths to packages to install, URLs or file paths").Strings()
```

Comment:
packages was the right terminology here in my opinion, just "URLs" wasn't .. I'd be ok with just a generic "package identifier"

Reply:
Thought experiment: we could use the [file URI notation](https://en.wikipedia.org/wiki/File_URI_scheme) to differentiate between `http[s]://...` (URL) and paths using `file://...`.

Then we could stick with the `packages` terminology:
```
installCmdURIs := installCmd.Arg("packages", "packages to install using URI notation. Currently, URLs and local file path URIs according to rfc8089 are supported.").Strings()
```